docs/introduction/GettingStarted.md

```diff
@@ -0,0 +1,156 @@
+applies to ReduxKotlin, however, there is also a lot of Javascript specific information that may
+_not_ apply to ReduxKotlin. This site documents how to get started with ReduxKotlin and provides 
+examples of using ReduxKotlin in projects. There are suggestions and examples, but ultimately the 
+core ReduxLibrary is not opinionated. How you use it in your project is up to you. This site will 
+continue to be updated as Kotlin multiplatform matures.
+
+
+## Installation
+
+ReduxKotlin is available on Maven central.  
+
+__For a multiplatform project:__
+
+Artifacts are published with Gradle meta-data, so the dependency only needs to be declared in your
+common sourceset:
+
+```groovy
+kotlin {
+    sourceSets {
+        commonMain {
+            dependencies {
+                implementation "org.reduxkotlin:redux-kotlin:0.2.6"
+            }
+        }
+    }
+}
+```
+
+Gradle meta-data is a preview feature, and will need to be turned on with 
+`enableFeaturePreview("GRADLE_METADATA")` in your gradle.settings file.
+
+__For single platform project (i.e. just Android):__
+
+```groovy
+dependencies {
+    implementation "org.reduxkotlin:redux-kotlin:0.2.6"
+}
+```
+
+For more details, see the [Installation](Installation.md) page.
+
+## Basic Example
+
+The whole state of your app is stored in an object tree inside a single _store_.  
+The only way to change the state tree is to emit an _action_, an object describing what happened.  
+To specify how the actions transform the state tree, you write pure _reducers_.
+
+That's it!
+
+```kotlin
+/**
+ * This is a reducer, a pure function with (state, action) -> state signature.
+ * It describes how an action transforms the state into the next state.
+ *
+ * The shape of the state is up to you: it can be a primitive, an array, an object, etc.
+ * Usually this will be a data class, because the copy method is useful for creating the new state.
+ * In this contrived example, we are just using an Int for the state.
+ *
+ * In this example, we use a `when` statement and type checking, but other methods are possible,
+ * such as a 'type' string field, or delegating the reduction to a method on the action objects.
+ */
+fun reducer(state: Int, action: Action) =
+    when (action) {
+        is Increment -> state + 1
+        is Decrement -> state - 1
+        else -> state
+    }
+
+/**
+ * Actions are plain objects that represent an action in the app. These can be 
+ * plain objects or data classes and have fields that hold data necessary for
+ * the reducer to update the state.
+ */
+class Increment
+class Decrement
+
+// Create a Redux store holding the state of your app.
+// 0 is the initial state
+val store = createStore(reducer, 0)
+
+// You can use subscribe() to update the UI in response to state changes.
+// Normally you'd use an additional layer or view binding library rather than subscribe() directly.
+
+store.subscribe { logger.debug(store.state)}
```

Comment:
We should document how to unsubscribe here. It's always a code smell whenever there's a subscribe without an unsubscribe. Warning bells for memory leaks immediately go off in my head.

Reply:
done